pkgs/development/python-modules/imgaug/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildPythonPackage, fetchPypi, numpy, scipy, scikitimage, opencv3, six }:
+
+buildPythonPackage rec {
+  pname = "imgaug";
+  version = "0.2.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wy8ydkqq0jrwxwdv04q89n3gwsr9pjaspsbw26ipg5a5lnhb9c2";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    scikitimage
+    opencv3
+    six
+  ];
+
+  # disable tests due to python 3 segmentation fault
```

Comment:
If there are no tests, that's what the comment should say.

Reply:
@dotlambda imgaug has tests, it just doesn't work due to python 3 segmentation fault. https://github.com/NixOS/nixpkgs/pull/43720#issuecomment-405842777

Reply:
> The current imgaug.tar.gz does not bundle any tests in the source archive so there are no tests to run anyway.

If I understand it correctly, there tests but not in the tarball. The comment could read something like `No tests in PyPI tarball`.
The segfault likely happens because the tests are missing.